colibris/docs/ui/index.html

```diff
@@ -0,0 +1,57 @@
+<!-- HTML for static distribution bundle build -->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Swagger UI</title>
+    <link rel="stylesheet" type="text/css" href="##STATIC_BASE_URL##/swagger-ui.css">
+    <link rel="icon" type="image/png" href="##STATIC_BASE_URL##/favicon-32x32.png" sizes="32x32"/>
+    <link rel="icon" type="image/png" href="##STATIC_BASE_URL##/favicon-16x16.png" sizes="16x16"/>
+    <style>
+        html {
+            box-sizing: border-box;
+            overflow: -moz-scrollbars-vertical;
+            overflow-y: scroll;
+        }
+
+        *,
+        *:before,
+        *:after {
+            box-sizing: inherit;
+        }
+
+        body {
+            margin: 0;
+            background: #fafafa;
+        }
+    </style>
+</head>
+
+<body>
+<div id="swagger-ui"></div>
+
+<script src="##STATIC_BASE_URL##/swagger-ui-bundle.js"></script>
+<script src="##STATIC_BASE_URL##/swagger-ui-standalone-preset.js"></script>
```

Comment:
Instead of this custom templating (`##STATIC_BASE_URL##` and `##API_SPEC_URL##`), all that is in `ui` is copy-paste build from latest version of swagger.

Reply:
It's good to hear that. We should probably link this static URL with those defined in routes. Just maybe always available, not only when DEBUG is true.

Reply:
@zalmanu, what do you think? Should we add those routes in template/skeleton? I would prefer to just have a setting (like the one we already have) to let the user set the path to documentation (maybe some option to disable it), but I wouldn't add those routes in the codebase.